src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        Circle circle = new Circle(new ColorSupplier().getRandomColor(), new Random()
+                .nextInt(100));
+        Square square = new Square(new ColorSupplier().getRandomColor(), new Random()
+                .nextInt(100));
+        Rectangle rectangle = new Rectangle(new ColorSupplier().getRandomColor(), new Random()
+                .nextInt(100), new Random().nextInt(100));
+        RightTriangle rightTriangle = new RightTriangle(new ColorSupplier().getRandomColor(),
+                new Random().nextInt(100), new Random().nextInt(100));
+        IsoscelesTrapezoid isoscelesTrapezoid = new IsoscelesTrapezoid(new ColorSupplier()
+                .getRandomColor(), new Random().nextInt(100), new Random()
+                .nextInt(100), new Random().nextInt(100));
+        Figure[] figures = new Figure[]
+                {isoscelesTrapezoid, circle, rectangle, rightTriangle, square};
+        return figures[new Random().nextInt(figures.length)];
```

Comment:
let's first randomly choose which shape we will create, and then create it, otherwise you create 4 shapes in vain every time

Reply:
also 100 is a magic number, let's make it a constant